api/bash/gitline

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+DOMAIN="https://api.github.com"
+source ~/projects/github_bash_project/.privateVariables.sh
+
+# Assigning Arguments
+whichMethod=$1
+username="$2"
+label_state_or_repo_name=$3
+is_private=$4
+
+function fetchIssuesWhereIAmAssignedWithLabel {
+  response=$(curl --silent ${DOMAIN}/search/issues?q=+label:${label_state_or_repo_name}+assignee:${username}+state:open&sort=created&order=asc)
```

Comment:
If we're going to do this on an enterprise instance, we need to supply a `TOKEN` into this, which is actually being brought in by source above. We should probably just attempt to read from a `.env` file in this repo (obviously a `.env.sample` would be set for showing people what to do)